util/mqtt.c

```diff
@@ -355,6 +355,26 @@ mqtt_init (const char *server_uri)
   return 0;
 }
 
+/**
+ * @brief Reinitializes communication after mqtt_reset was used
+ *
+ */
+void
+mqtt_reinit ()
+{
+  const char *server_uri;
+  // init new global client if it was mqtt_reset()
```

Comment:
Sorry for being annoying but either remove this comment or move it out of the function.

Reply:
It is not annoying at all. It is important to keep the code clean and I am glad that you noticed it!